contentScript.js

```diff
@@ -28,6 +28,49 @@ function addingCssElement(elementId, status, numRelevantFiles) {
 	}
 };
 
+
+function addCssElementToBitbucket(highlightedPRIds) {
+
+	// To do : remove this setTime out method once data is coming from api 
+	setTimeout(() => {
+		let tables = document.getElementsByTagName('table');
```

Comment:
Use `const` instead of `let` wherever possible. (You don't need `let` unless you are REASSIGNING to a particular token)